### DISCUSSION TITLE
Some types are shown as unresolved in my GAS project unexpectedly

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [google/clasp#1005] [Q&A]
[Here](https://github.com/EmilyGraceSeville7cf/javascript-presentation-generator/tree/main) is my project. Sometimes when I hover on types defined in types/*.js files as JSDoc comments I get /* unresolved */ warning from intellisence. Why it happens and how to fix it?

![image](https://github.com/google/clasp/assets/42812113/7e9b5f89-a0f5-4b3a-a553-2de259b26632)

[Open original question](https://groups.google.com/g/google-apps-script-community/c/kIw_yMBr1Xs/m/WNiJlukWAwAJ)

**Comment by Nu11u5** [comment]
I found that the file containing the JSDoc has to be open in an editor tab for it to be parsed.  If there is a way around this so the whole project is automatically parsed I would like to know.
